### PR TITLE
Update Run a Node section

### PR DESCRIPTION
**Pull request by eshaben**
This PR changes the name of the "Compile the Binary" page to "Using the Binary" and adds a section to cover purging binary data. Update the rest of the docs where needed to point to this new section.

This PR addresses the following:
- fixes a bug in `renderSubsection.js`
- Separate docker and systemd installation instructions to new pages
- moves Compile the Binary to the systemd page
- removes any unused images from telemetry and compile the binary
- adds new banner images/index page icons for docker and systemd pages
- adds purge chain commands to the docker and systemd pages, and the moonbeam-dev page
- updates the Updating the Client section for both docker and systemd pages

### CODE REVIEW

File: builders/get-started/moonbeam-dev.md [hunk 1]
```diff
@@ -166,9 +166,11 @@ When running a node via the binary file, data is stored in a local directory typ
 ./target/release/moonbeam purge-chain --dev -y
 ```
 
-This will remove the data folder, note that all chain data is now lost.
+This will remove the data folder, note that all chain data is now lost. To learn more about all of the available `purge-chain` commands, you can checkout the [Purging Binary Data](/node-operators/networks/compile-binary/#purging-binary-data) section of our documentation.
+
+!!! note
+    If you used Docker, the data folder is related to the Docker container itself.
```
Comment (albertov19) [inline builders/get-started/moonbeam-dev.md:172]: I don't think this is entirely true, when I run my Moonbase Alpha node I specify a volume where the data is stored:

```
-v "/var/lib/alphanet-data:/data"
```

Not sure what is the default if this is not provided

**Review by albertov19 — CHANGES_REQUESTED** [review]
So what about if I'm a node operator and I want to purge my chain?

In that code snippet I do see the stuff about Moonbase Alpha will be ocasionally purged (which does not seem to be the case). Can you change that part?

I also made a minor comment

File: builders/get-started/moonbeam-dev.md (hunk 1 above)
Comment (eshaben) [reply builders/get-started/moonbeam-dev.md:172]: > So what about if I'm a node operator and I want to purge my chain?

I'm not sure what you mean by this? What is the alternative to being a node operator?

> In that code snippet I do see the stuff about Moonbase Alpha will be ocasionally purged (which does not seem to be the case). Can you change that part?

What code snippet are you talking about?

Comment (eshaben) [reply builders/get-started/moonbeam-dev.md:172]: I didn't realize you could specify chain data using Docker - I'll update the docs for the Docker section so people know they can do that

**Comment by eshaben** [comment]
Ah bummer... I was so focused on getting the systemd part of this correct that I definitely overlooked and made assumptions on the Docker side of things. That is my bad, I apologize 😔 

I will be sure to make no assumptions and leave no stone unturned moving forward.

In case anyone is interested about Docker stuff:
```
By default all files created inside a container are stored on a writable container layer. This means that:

The data doesn’t persist when that container no longer exists, and it can be difficult to get the data out of the container if another process needs it.
A container’s writable layer is tightly coupled to the host machine where the container is running. You can’t easily move the data somewhere else.
Writing into a container’s writable layer requires a storage driver to manage the filesystem. The storage driver provides a union filesystem, using the Linux kernel. This extra abstraction reduces performance as compared to using data volumes, which write directly to the host filesystem.
Docker has two options for containers to store files in the host machine, so that the files are persisted even after the container stops: volumes, and bind mounts. If you’re running Docker on Linux you can also use a tmpfs mount. If you’re running Docker on Windows you can also use a named pipe.
```
Source: https://docs.docker.com/storage/

**Comment by eshaben** [comment]
Thanks for reviewing this! Just not sure about your node operator comment... other than that I have addressed the other comments you made and this is ready for a review again

**Review by albertov19 — CHANGES_REQUESTED** [review]
Section index pages logos missing 😜

File: builders/get-started/moonbeam-dev.md (hunk 1 above)
Comment (albertov19) [reply builders/get-started/moonbeam-dev.md]: When running the docker command you can specify where the data is stored, I'm not sure what happens when you don't specify anything.

Comment (eshaben) [reply builders/get-started/moonbeam-dev.md]: this is outdated, thats been fixed 😄

Comment (eshaben) [reply builders/get-started/moonbeam-dev.md]: just about to push up the changes for the reorg but I like to review it in the morning now so I dont make little mistakes that can be easily avoided with a fresh mind haha

File: node-operators/networks/run-a-node/docker.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+```
+sudo docker stop `CONTAINER_ID`
+```
+
+Then, install the new version by repeating the steps described before, making sure that you are using the latest tag available. After updating, you can start the service again.
+
+### Purging the Chain {: #purging-the-chain } 
+
+On an as-needed basis, Moonbase Alpha might be purged and reset. If a purge is required, node operators will be notified in advance (via our [Discord channel](https://discord.gg/PfpUATX)). You can also purge your node if your individual data directory becomes corrupted.
```
Comment (eshaben) [inline node-operators/networks/run-a-node/docker.md:168]: Remove on an as-needed basis, make this section more about general purging versus a testnet reset

File: node-operators/networks/run-a-node/docker.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+As Moonbeam development continues, it will sometimes be necessary to upgrade your node software. Node operators will be notified on our [Discord channel](https://discord.gg/PfpUATX) when upgrades are available and whether they are necessary (some client upgrades are optional). The upgrade process is straightforward and is the same for a full node or collator.
+
+First, stop the docker container:
+
+```
+sudo docker stop `CONTAINER_ID`
+```
+
+Then, install the new version by repeating the steps described before, making sure that you are using the latest tag available. After updating, you can start the service again.
+
```
Comment (eshaben) [inline node-operators/networks/run-a-node/docker.md:165]: expand on this section in a way that points the user in the right direction for restarting their node aside from just saying you can start service again

Comment (eshaben) [reply node-operators/networks/run-a-node/docker.md:165]: check systemd page too!

**Comment by eshaben** [comment]
Alright this is ready for review again

**Review by albertov19 — APPROVED** [review]
Made a minor modification but looks good!

**Pull request merged**